app/src/config.js

```diff
@@ -108,27 +109,24 @@ export default {
       logoHover: '',
       logoLight: '',
       logoDark: '',
+      hasLightLogo: false,
       showOnModal: true,
     },
-    [REDDIT_VERIFIER]: {
-      description: '',
-      typeOfLogin: REDDIT,
-      name: REDDIT,
-      clientId: VUE_APP_REDDIT_CLIENT_ID,
-      logoHover: '',
-      logoLight: '',
-      logoDark: '',
-      showOnModal: true,
-    },
-    [TWITCH_VERIFIER]: {
+    [TWITTER_VERIFIER]: {
       description: '',
-      typeOfLogin: TWITCH,
-      name: TWITCH,
-      clientId: VUE_APP_TWITCH_CLIENT_ID,
+      typeOfLogin: TWITTER,
+      name: TWITTER,
+      clientId: VUE_APP_TWITTER_CLIENT_ID,
       logoHover: '',
       logoLight: '',
       logoDark: '',
+      hasLightLogo: false,
       showOnModal: true,
+      jwtParameters: {
+        domain: LOGIN_DOMAIN,
+        connection: 'twitter',
+        isVerifierIdCaseSensitive: false,
+      },
```

Comment:
Use flags for mobile, desktop separately which enable their visibility

Reply:
added showOnDesktop, showOnMobile